packages/test/src/integration-tests-old.ts

```diff
@@ -677,6 +677,20 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     );
   });
 
+  test('Workflow can upsert memo', async (t) => {
+    const { client } = t.context;
+    const workflow = await client.start(workflows.upsertAndReadMemo, {
+      taskQueue: 'test',
+      workflowId: uuid4(),
+      args: [{ note2: 'bar' }],
```

Comment:
Please use the newer `test-integration-workflows.ts` and test adding, updating and removing.